planet-dumper.py

```diff
@@ -62,14 +64,22 @@ def load_env():
             object_strorage_config['bucket_name'] = os.environ['OBJECT_STORAGE_BUCKET']
             object_strorage_config['should_use_ssl'] = os.getenv('OBJECT_STORAGE_USE_SSL', False)
             object_strorage_config['verify_root_cert'] = os.getenv('OBJECT_STORAGE_VERIFY_ROOT_CERT', False)
```

Comment:
better to convert it to bool, in the following lines you are comparing it to `'true'` while the default value you set is `False`

let's stick to bool
```python
def convertStringToBool(s):
  return s.lower() in ['true', '1', 't', 'y', 'yes', 'yeah', 'yup', 'certainly', 'uh-huh']
```